cypress/e2e/live.cy.ts

```diff
@@ -0,0 +1,329 @@
+        .find('h4')
+        .contains('Time Details')
+        .next('ul')
+        .children('li')
+        .as('timeDetails');
+      cy.get('@timeDetails')
+        .eq(0)
+        .children('p')
+        .should('contain.text', format(edit.startDate, 'PP'));
+      cy.get('@timeDetails').eq(1).children('p').should('contain.text', format(edit.endDate, 'PP'));
+      cy.get('@timeDetails')
+        .eq(2)
+        .children('p')
+        .should('contain.text', `${format(edit.startDate, 'p')} - ${format(edit.endDate, 'p')}`);
+      cy.get('@timeDetails')
+        .last()
+        .children('p')
+        .should(
+          'contain.text',
+          edit.activeDays
+            .map((value) => ActiveDaysOptions.find((option) => option.value === value)?.label)
+            .join(', ')
+        );
+    });
+  });
+
+  /**
+   * Verify Duplicate Session Flow
+   */
+
+  it('Verify Duplicate Session Flow', () => {
+    cy.get('@sessionId').then((sessionId) => {
+      cy.get('table > tbody > tr').each(($tr) => {
+        // Find the created session using sessionId
+        if ($tr.children('td').eq(0).text().trim() === String(sessionId)) {
+          const tdLength = $tr.children('td').length;
+
+          // Click on edit
+          cy.wrap($tr)
+            .children('td')
+            .eq(tdLength - 1)
+            .find('button')
+            .focus()
+            .type('{enter}');
+
+          cy.get('div[data-radix-menu-content]')
+            .should('be.visible')
+            .find('a[role="menuitem"]')
+            .contains('Duplicate')
+            .should('exist')
+            .click();
+
+          // Verify the edit session page
+          cy.url().should('include', '/session/duplicate');
+          cy.url().should('include', `id=${sessionId}`);
+
+          // Fill basic details
+          cy.get('input[name="name"]').should('be.empty');
+          cy.customInput('name', duplicate.name);
+          cy.get('button').contains('Next').click();
+
+          cy.customInput('platformLink', duplicate.platformLink);
+          cy.get('input[name="platformId"]').should('have.value', duplicate.platformId);
+
+          // Fill platform details
+          cy.get('button').contains('Next').click();
+
+          // Fill timeline details
+          cy.customDatePicker('startDate', duplicate.startDate);
+          cy.customDatePicker('endDate', duplicate.endDate);
+
+          // TODO: uncomment this to create duplicate session
+          // cy.get('button').contains('Next').click();
+          // cy.url().should('include', '/');
+          // cy.get('table > tbody > tr')
+          //   .eq(0)
+          //   .children('td')
+          //   .then(($tds) => {
+          //     const tdsArray = $tds.toArray();
+          //     const hasName = tdsArray.some((td) => td.innerText.trim() === duplicate.name);
+          //     expect(hasName).to.be.true;
+          //   });
+        }
+      });
+    });
+  });
+
+  afterEach(function () {
```

Comment:
write one condition where one of the stages is incompletely filled.. so "Next" button is disabled

Reply:
maybe just before filling last item in a stage. 
check that next is disabled
then, fill the last item
then, check that next is enabled.

Reply:
Added